src/kudu/consensus/consensus.proto

```diff
@@ -419,6 +419,9 @@ message ConsensusRequestPB {
   // Must be set if this request was proxied on behalf of the leader.
   optional bytes proxy_caller_uuid = 13;
 
+  // Hop count / TTL field for proxy requests.
+  optional int32 proxy_hops_remaining = 14;
```

Comment:
nit: TTL is a unique usage for hop count.

Reply:
I don't understand this comment. Are you saying I should remove this reference to TTL? I just thought it helped clarify the meaning of hop count in this context.

Reply:
nit suggestion was to not use TTL since it can confuse with other usages of TTL in databases which are based on time, and instead use the simpler terminology of max hops.